### PR TITLE
Njbrake/profile styles

### PR DESCRIPTION
**Pull request by njbrake**
This pull incorporates several baseline changes to support theming as the app builds out. Fonts have been added, the blob svg has been added, and basic theming colors have been updated. Most UI improvements were performed only on Profile Header.

Also added auth screens and notifications tab.

image

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -19,6 +19,31 @@ class App extends React.Component {
 	state = {
 		isLoadingComplete: false,
 	};
+	_loadResourcesAsync = async () => {
+		return Promise.all([
+			Font.loadAsync({
+				// This is the font that we are using for our tab bar
+				...Icon.Ionicons.font,
```
Comment (coopersamuel) [inline App.js]: We'll eventually use the svgs that Bek draws, so this is temporary. But fine for now :)

File: App.js [hunk 2]
```diff
@@ -19,6 +19,31 @@ class App extends React.Component {
...
+				'enriq-medium': require('./assets/fonts/ENRIQMedium.ttf'),
+				'enriq-Black': require('./assets/fonts/ENRIQBlack.ttf'),
+				'enriq-light': require('./assets/fonts/ENRIQLight.ttf'),
+			}),
+		]);
+	};
+
+	_handleLoadingError = error => {
+		// In this case, you might want to report the error to your error
+		// reporting service, for example Sentry
```
Comment (coopersamuel) [inline App.js]: When we have something like this, we'll abstract it out into it's own service

File: App.js [hunk 3]
```diff
@@ -19,6 +19,31 @@ class App extends React.Component {
 	state = {
 		isLoadingComplete: false,
 	};
+	_loadResourcesAsync = async () => {
+		return Promise.all([
+			Font.loadAsync({
+				// This is the font that we are using for our tab bar
+				...Icon.Ionicons.font,
+				// We include SpaceMono because we use it in HomeScreen.js.
```
Comment (coopersamuel) [inline App.js]: This comment is outdated I believe

File: App.js (hunk 1 above)
Comment (njbrake) [reply App.js]: Not quite sure what you mean. This code snippet is the part of the app where we are initially loading up the app and prefetching the fonts.

**Pull request merged**